transport/grpc/handler.go

```diff
@@ -149,15 +149,15 @@ func (h *handler) handleStream(
 	stream := newServerStream(ctx, &transport.StreamRequest{Meta: transportRequest.ToRequestMeta()}, serverStream)
 	tServerStream, err := transport.NewServerStream(stream)
 	if err != nil {
-		return err
+		return transport.UpdateSpanWithErr(span, err, 2)
```

Comment:
Can't we get yarpc status code from err without in `transport.UpdateSpanWithErr` without passing the integer?
```
yarpcerrors.FromError(err).Code()
```